### PR TITLE
Migrating covent documentation into repository as plain text

### PR DESCRIPTION
**Pull request by martukas**
I don't even know where to start.

Pretty much everything from Covent docs:
* 1.1 Design overview
* 1.2 Status update
* 3.2 Assembly instructions
* 5.1 Production methods of custom components

The PDFs have been removed and instead all content has been put in various locations around the repository.

Of particular note:
* Statement of open-source philosophy linked from main readme.
* All of pneumatics system design. I have sweeped out Ethan's elaborate tree of directories with empty stubs. For now the pneumatics design is a single file, but if someone wants to extract it into a proper hierarchy, they can do that later.
* Content for mechanical and electrical systems design pages.
* More software architecture documentation.
* Most sub-assemblies now have a design motivation section and section on fabrication at scale.
* Pizza build has detailed instructions with vivid illustrations.
* Enclosed build has a section on enclosure with drawings.
* Testing page has content on SIL, HIL and lifetime leader testing.
* Page on Modelon under design.

A lot of this can probably use proofreading, but I don't expect any of you to get that anal about it. Just make sure the content ended up in the right places and there are no blatant lies.

### CODE REVIEW

**Review by tapemaster — APPROVED** [review]

File: design/electrical-system/README.md [hunk 1]
```diff
@@ -1,19 +1,159 @@
...
+all time-critical operations, such as ventilator pneumatic control, are
+handled by the STM32.
+
+A serial communication link using checksums to ensure data integrity
+provides reliable communication of settings and data between the two
+computing elements.
+
+In the event of a system crash or failure in the UI Computer, the Cycle
+Controller will continue to operate the ventilator with the most recent
+valid settings received from the UI Computer.
```
Comment (asmodai27) [inline design/electrical-system/README.md:64]: Though it makes sense to have some elements of software behaviour in the electrical design, I think this paragraph is a bit less relevant to electrical, as it describe something that is mostly software (admittedly enabled by a hardware choice).
I guess it feels redundant to me with the software architecture discussion, and we could just say the dual-computer design allows more flexibility and robustness in the software design, and link to the SW architecture file, to limit redundancy?

**Review by asmodai27 — APPROVED** [review]
One minor remark, but this is very nice as is already!

File: design/electrical-system/README.md (hunk 1 above)
Comment (martukas) [reply design/electrical-system/README.md]: Thanks. Good catch! Removed this as indeed it is redundant.

File: quality-assurance/README.md [hunk 2]
```diff
@@ -12,4 +22,8 @@ Documentation relating to the testing equipment, strategy and test data is in th
 about regulatory documentation.
```
Comment (paulgun89) [inline quality-assurance/README.md:22]: Links to the notes about the presentation, not the presentation itself.

File: design/README.md [hunk 3]
```diff
@@ -1,48 +1,75 @@
...
--------------
-
-This document contains description of ventilator design implementation and closure of issues which
+This section of the repository contains the description of ventilator design implementation and closure of issues which
 apply at the system level, such as architecture configuration.
 
-## Overview
+## Design Motivations
 
-A ventilator is a device which supports the breathing of a patient who cannot generate
```
Comment (paulgun89) [inline design/README.md:10]: Rephrase to be concise:

Ventilators are devices that support patient breathing. The device must generate sufficient pressure to overcome lung and machine resistance. Pressure must vary over time as the lungs fill and then deflate. To improve oxygenation, ventilators also almost universally apply some back-pressure to the lungs (PEEP).

There are dozens of ways to design a ventilator and each has trade-offs. This document describes the design decisions and reasoning that led to the final design implementation.

The first design decision to address is how the ventilator should generate the pressure required to drive air into the lungs.

Displacement type devices provide a mechanical compression of a fixed volume to drive air into the lungs. These are commonly employed in existing ventilators, and they provide a high degree of volumetric accuracy and precise control of flow. There are two challenges here. The first is a question of supply chain. The volume of air which must be delivered into a lung is relatively large, as far as traditional cylinders go. The lubricant and gasket material must be oxygen compatible, and relatively expensive and sensitive drive mechanisms are required. The designs end up rather heavy and expensive. Bellows-type devices can be helpful in addressing the volume issue, but are very challenging to design for the millions of cycles required, in addition to being difficult to source.

File: design/conop.md [hunk 4]
```diff
@@ -40,4 +40,4 @@ Exhale valve moves to exhale position (fully open)
 Controlled adjusts pressure set point for closed loop control to PEEP pressure - controller adjusts blower speed to meet PEEP.
```
Comment (paulgun89) [inline design/conop.md:40]: The breath control cycles here are listed as run-on sentences. Separate into bullet points?

File: design/README.md [hunk 5]
```diff
@@ -1,48 +1,75 @@
 # System Architecture
```
Comment (paulgun89) [inline design/README.md:1]: This is called "System Architecture" on its own page but "System Design" in the table of contents. Let's match titles!

File: manufacturing/enclosed_build/README.md [hunk 6]
```diff
@@ -1,21 +1,48 @@
 # Enclosed build
```
Comment (paulgun89) [inline manufacturing/enclosed_build/README.md:2]: Put a disclaimer here about not using on patients yet like is on the Pizza Build page.

File: manufacturing/venturi/README.md [hunk 7]
```diff
@@ -1,30 +1,60 @@
...
+
+## Design basis
 
 The venturi is used to measure flow by measuring a change in pressure as the flow accelerates through the throat.
 Relative to other pressure sensor types, a venturi flow meter produces the largest signal pressure at the smallest
 pressure loss. This allows significantly cheaper and more widely available pressure sensors to be used, because a
-signal at peak flow can be 5 to 15 times larger than the flow resistance.
+signal at peak flow can be 5 to 15 times larger than the flow resistance. For a resistance type flow sensor, the ratio
+is 1:1. The change in pressure is measured between the two ports, and this is correlated to the flow rate through the
+orifice. It is important to note that the as the flow expands, almost all of this pressure is recovered to the original
```
Comment (paulgun89) [inline manufacturing/venturi/README.md:14]: typo: that **the** as the flow expands

**Review by paulgun89 — APPROVED** [review]

**Pull request merged**